### PR TITLE
SEA-346: New heat pump measures

### PR DESCRIPTION
**Pull request by laurasaunders1**
Code updates to support the new air and source heat pump recommendations that the BRE API now returns.

### CODE REVIEW

File: user-site-server/src/main/resources/db/changelogs/2022-05-05-add-heat-pump-grants.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+SET @p_1 = LAST_INSERT_ID();
+
+INSERT INTO `wp_postmeta`
+(post_id, meta_key, meta_value)
+VALUES
+(@p_1,'heading','Boiler upgrade scheme'),
+(@p_1,'_heading','field_5a0d969b5dbb1'),
+(@p_1,'description','Get £6000 off the cost of your heat pump and installation. Further detail is <a href="https://www.gov.uk/guidance/check-if-you-may-be-eligible-for-the-boiler-upgrade-scheme-from-april-2022">here</a>.'),
+(@p_1,'_description','field_5a0d96a17c14c'),
+(@p_1,'link_to_measures','0'),
```
Comment (malinaslevoaca1) [inline user-site-server/src/main/resources/db/changelogs/2022-05-05-add-heat-pump-grants.xml:30]: we need to update this in WP, right?

**Review by malinaslevoaca1 — APPROVED** [review]
Looks great!! 👏

File: user-site-server/src/main/resources/db/changelogs/2022-05-05-add-heat-pump-grants.xml (hunk 1 above)
Comment (laurasaunders1) [reply user-site-server/src/main/resources/db/changelogs/2022-05-05-add-heat-pump-grants.xml:30]: Yep! I had a look at capturing the ID's and linking the other existing measure, but it seemed a bit more hassle and error prone than it was worth!

The measures I have added are also blank, but adding them via migration gave better control over the names/ was a bit tidier!

Comment (malinaslevoaca1) [reply user-site-server/src/main/resources/db/changelogs/2022-05-05-add-heat-pump-grants.xml:30]: Makes sense, thanks 😄

**Review by malinaslevoaca1 — APPROVED** [review]

**Pull request merged**